users/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+
+class User(models.Model):
+    full_name    = models.CharField(max_length=40)
+    email        = models.EmailField(max_length=50, unique=True)
+    password     = models.CharField(max_length=100)
+    membership   = models.BooleanField(default=False)
+    address      = models.CharField(max_length=255)
+    phone_number = models.CharField(max_length=20)
+    gender       = models.CharField(max_length=10)
+
+    created_at   = models.DateTimeField(auto_now_add=True)
+    updated_at   = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        db_table = 'users'
+
+
+class Ordered_state(models.Model):
+    state = models.CharField(max_length=30)
+
+    class Meta:
+        db_table = 'ordered_states'
```

Comment:
주문상태는 order status라는 말이 더 잘 어울리니 아래와 같이 수정해주세요!
```suggestion
class OrderStatus(models.Model):
    status = models.CharField(max_length=30)

    class Meta:
        db_table = 'order_status'
```